lib/metrics/log.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const normalizeName = require('./utils').normalizeName;
+const formatLabels = require('./utils').formatLabels;
+
+// Metric logger implementation
+class LogMetric {
+    constructor(options, client) {
+        this.options = options;
+        this.client = client;
+    }
+
+    _call(func, value, labels) {
+        labels = formatLabels(this.options, labels);
+        func.apply(this.client, [normalizeName(labels.join('.')), value]);
+    }
+
+    increment(amount, labels) {
+        this._call(this.client.increment, amount, labels);
+    }
+
+    decrement(amount, labels) {
+        this._call(this.client.decrement, amount, labels);
+    }
+
+    observe(value, labels) {
+        this._call(this.client.gauge, value, labels);
+    }
+
+    gauge(amount, labels) {
+        this._call(this.client.gauge, amount, labels);
+    }
+
+    set(value, labels) {
+        this._call(this.client.gauge, value, labels);
+    }
+
+    timing(value, labels) {
+        this._call(this.client.timing, value, labels);
+    }
+
+    endTiming(startTime, labels) {
+        this.timing(Date.now() - startTime, labels);
+    }
+}
+
+// A simple console reporter. Useful for development.
+class LogClient {
+    constructor(options, logger) {
+        this._logger = logger;
+        this._serviceName = options.name ? `${options.name}.` : 'service-runner.';
+        this.methods = options.methods;
```

Comment:
What is methods supposed to be for log client?

Reply:
Great catch!  Somehow this piece got missed.  I just pushed a fix.